koku/masu/api/running_celery_tasks.py

```diff
@@ -71,18 +71,26 @@ def clear_celery_queues(request):
     if request.method == "GET":
         params = request.query_params
         clear_all = params.get("clear_all")
-        LOG.info(f"Clearing all queues parameter: {clear_all}")
+        queue = params.get("queue")
 
         # clear default_celery queue
         purged_tasks = app.control.purge()
```

Comment:
I presume if we want to clear a specific queue, we don't want to run this?

Reply:
So this one actually only clears the celery queue. Honestly I'm not sure why we even have this, I feel like it does nothing. But at the same time this one gets run when you just hit `http://localhost:5042/api/cost-management/v1/clear_celery_queues/` regardless of whether you pass in `?clear_all` or another queue.

Reply:
I think maybe we just remove this line, since its highly unlikely that we want to clear the celery queue ever anyway.